yt_dlp/extractor/nate.py

```diff
@@ -0,0 +1,159 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    traverse_obj,
+    unified_strdate,
+)
+
+
+class NateIE(InfoExtractor):
+    _VALID_URL = r'https?://tv\.nate\.com/clip/(?P<id>[0-9]+)'
+
+    _TESTS = [{
+        'url': 'https://tv.nate.com/clip/1848976',
+        'info_dict': {
+            'id': '1848976',
+            'ext': 'mp4',
+            'title': '[결승 오프닝 타이틀] 2018 LCK 서머 스플릿 결승전 kt Rolster VS Griffin',
+            'description': 'md5:e1b79a7dcf0d8d586443f11366f50e6f',
+            'thumbnail': 'http://image.pip.cjenm.com/CLIP/GA/B120189687/B120189687_EPI0056_03_B.jpg',
+            'upload_date': '20180908',
+            'age_limit': 15,
+            'duration': 73,
+            'uploader': '2018 LCK 서머 스플릿(롤챔스)',
+            'channel': '2018 LCK 서머 스플릿(롤챔스)',
+            'channel_id': 3606,
+            'uploader_id': 3606,
```

Comment:
aren't they expected to be string value? :thinking:

Reply:
yup.
Already fixed locally.
But i do have doubt regarding `tags`, is it safe to just use `'tags': list` 🤔